api/definition/swagger.yaml

```
@@ -0,0 +1,273 @@
+openapi: 3.0.0
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/ANDREWJEVANS/fingertips-next-api/1.0.0
+info:
+  description: An API to query public health indicator data.
+  version: "1.0.0"
+  title: Fingertips API
+  contact:
+    email: you@your-company.com
+tags:
+  - name: indicators
+    description: Endpoints dealing with public health indicators
+paths:
+  /indicators:
+    get:
+      tags:
+        - indicators
+      summary: searches indicators
+      operationId: searchIndicators
+      description: |
+        By passing in the appropriate options, you can search for
+        public health indicators
+      parameters:
+        - $ref: "#/components/parameters/indicator_ids"
+      responses:
+        '200':
+          description: search results matching criteria
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/IndicatorSummary'
+        '400':
+          $ref: "#/components/responses/BadRequest"
+        '500':
+          $ref: "#/components/responses/InternalServerErrror"
+  /indicators/{indicator_id}:
+    get:
+      tags:
+        - indicators
+      summary: Get indicator
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Indicator"
+          description: The request was successful, and the server has returned the
+            requested resource in the response body.
+        '404':
+          $ref: "#/components/responses/NotFound"
+        '500':
+           $ref: "#/components/responses/InternalServerErrror"
+      parameters:
+        - $ref: "#/components/parameters/indicator_id"
+      description: >
+        Fetches details of a specific indicator by its unique identifier. The
+        response includes the indicator's metadata
+      operationId: getIndicator
+  /indicators/data:
+    get:
+      tags:
+        - indicators
+      summary: Get health data for an indicator
```

```suggestion
      summary: Get health data for indicators
```
As this supports retrieving data for multiple indicators.